vue-frontend/src/components/partials/ScreenHeaderV2.vue

```diff
@@ -1,43 +1,51 @@
 <template>
-  <div class="nav-container" :style="{ height: navContainerHeight + 'px' }">
+  <div
```

Comment:
![Screenshot from 2022-08-26 20-10-58](https://user-images.githubusercontent.com/94360312/186930232-2b1c06ff-1c9a-4549-b0df-bedbc9279c57.png)

![Screenshot from 2022-08-26 20-10-00](https://user-images.githubusercontent.com/94360312/186930278-b8c1978d-6761-485f-aac4-df800425bfe3.png)

The header does not look like the live website header